addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/handler/SqueezeBoxServerHandler.java

```diff
@@ -421,7 +421,8 @@ public void run() {
                     // Message is very long and frequent; only show when running at trace level logging
                     logger.trace("Message received: {}", message);
 
-                    if (message.startsWith("listen 1")) {
+                    // Fix for some third-party apps that are sending "subscribe playlist"
+                    if (message.startsWith("listen 1") || message.startsWith("subscribe playlist")) {
```

Comment:
Why not add some input validation to `SqueezeServerListener.handlePlaylistMessage` (i.e. length of `messageParts`)? That would fix the issue for any invalid message and not just this "subscribe playlist" message.

Reply:
Yeah, that was my first inclination. But, this is a `subscribe` command, not a `playlist` command, so I didn't want to deal with it inside `handlePlaylistMessage`. It was only a coincidence (the second argument was `playlist`) that it was being processed in `handlePlaylistMessage`.

Having said that, there's certainly no harm in being more defensive in `handlePlaylistMessage` (although with the LMS at end-of-life  for the last 5+ years, the likelihood of something changing is very low).

I'll push a commit that adds bounds checking in handlePlaylistMessage, in addition to the above change.

Reply:
Bounds checking was added.